src/Microsoft.Azure.WebJobs.Extensions.Kafka/Listeners/KafkaListener.cs

```diff
@@ -65,8 +74,15 @@ internal class KafkaListener<TKey, TValue> : IListener
             this.requiresKey = requiresKey;
             this.logger = logger;
             this.cancellationTokenSource = new CancellationTokenSource();
+            this.consumerGroup = string.IsNullOrEmpty(this.listenerConfiguration.ConsumerGroup) ? "$Default" : this.listenerConfiguration.ConsumerGroup;
+            this.topicName = this.listenerConfiguration.Topic;
+            this.functionId = functionDescriptor.Id;
+            this.scaleMonitorDescriptor = new ScaleMonitorDescriptor($"{functionId}-kafka-{topicName}-{consumerGroup}".ToLower());
```

Comment:
nit: change "kafka" to "kafkatrigger". All the other extensions follow that pattern